tests/builds/zika.t

```diff
@@ -64,7 +64,7 @@ Build a tree from the multiple sequence alignment.
   >  --method iqtree \
   >  --tree-builder-args "-seed 314159" > /dev/null
 
-  $ python3 "$TESTDIR/../../scripts/diff_trees.py" "results/tree_raw.nwk" "$TMP/out/tree_raw.nwk" --significant-digits 5
+  $ python3 "$TESTDIR/../../scripts/diff_trees.py" "results/tree_raw.nwk" "$TMP/out/tree_raw.nwk" --significant-digits 3
```

Comment:
Looking at these data again, 4 significant digits would be more appropriate than 3 or 5. The Zika genome is ~10,000 nucleotides long, so a scale of 1 / 10000 (or 0.0001 or 4 significant digits) would be appropriate for these tests.

Reply:
Fixed: https://github.com/nextstrain/augur/pull/796/commits/e8fde5c0ef5e6fe0cef022dc87ca113346be8c83